trending_data/src/main/java/com/google/sps/data/Country.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+public class Country {
+  String name = "";
+  public String alpha2Code = "";
+  double lat;
+  double lng;
+  // Given a String in this format:
+  // "Country","Alpha-2 code","Alpha-3 code","Numeric code","Latitude (average)","Longitude
+  // (average)"
+  // , creates the Country object which contains country name , alpha-2 code and the  lat,lng
+  public Country(String countryInfo) {
+
+    // will fail if country name has a (") inside it
+    ArrayList<String> fields = parse(countryInfo);
+
+    this.name = fields.get(0);
+    this.alpha2Code = fields.get(1);
+    this.lat = Double.parseDouble(fields.get(4));
+    this.lng = Double.parseDouble(fields.get(5));
+  }
+
+  // gets the info out of the string in the above format
+  private static ArrayList<String> parse(String countryInfo) {
+    ArrayList<String> fields = new ArrayList<String>();
+    // true if the index (i) is inside a pair of ("")
```

Comment:
I think it's better to create map instead of array to get value by parameters name, not by index

Reply:
Yeah I think having hard coded numbers in the code doesn't look good, I will put final variables that tell which index is  for which value. Having a map I think will over complicate things because then I will have to have strings for each parameter name plus it will slightly decrease efficiency also I will make the variables private because I will not use them in any other class